data/officialrajdeepsingh.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "Rajdeep Singh",
+  "type": "personal",
+  "bio": "JavaScript || Reactjs || Nextjs || Python || Rust || Biotechnology || Bioinformatic || Front-end Developer || Author || Open source contributor",
+
+  "tags": ["Open Source", "Javascript", "ReactJS", "Nextjs"],
+  "socials": [
+    { "icon": "FaGithub", "url": "https://github.com/officialrajdeepsingh" },
+    {
+      "icon": "FaLinkedin",
+      "url": "https://www.linkedin.com/in/officalrajdeepsingh/"
+    },
+    {
+      "icon": "FaTwitter",
+      "url": "https://twitter.com/Official_R_deep"
+    }
+  ],
+  "links": [
+    {
+      "name": "GitHub: Let's work on some projects",
+      "url": "https://github.com/officialrajdeepsingh",
+      "icon": "FaGithub"
+    },
+    {
+      "name": "LinkedIn: Let's connect",
+      "url": "https://www.linkedin.com/in/officalrajdeepsingh/",
+      "icon": "FaLinkedin"
+    },
+    {
+      "name": "Freecodecamp",
+      "url": "https://www.freecodecamp.org/news/author/officialrajdeepsingh/",
+      "icon": "FaFreeCodeCamp"
+    },
+    {
+      "name": "website: Let's connect",
+      "url": "https://www.linkedin.com/in/officalrajdeepsingh/",
+      "icon": "FaGlobe"
```

Comment:
Looks good, but it seems like you've added the same link twice. You might want to add your website link here.

Reply:
Now fix the issue